templates/default/user/move_extra_dirs.sh

```diff
@@ -15,9 +15,10 @@ then
     else
         mkdir -p "$extra_dir"
     fi
-    chown -R {{ user.uidnumber }}:{{ user.gidnumber }} "$extra_dir"
 fi
 
+chown -R {{ user.uidnumber }}:{{ user.gidnumber }} "$extra_dir"
```

Comment:
one issue is we gonna change ownership even if not necessary, which depending on content can be quite long.
should change only if gid is different

Reply:
https://github.com/genouest/genouestaccountmanager/pull/177/commits/f6096e2c209398cd4de0562b2c1313c9eacbe665